StatOverview/StatOverviewTab.lua

```diff
@@ -239,7 +239,7 @@ end
 function StatOverviewTab:EnableChatSendButton()
 	self.chatSendEnabled = true;
 	
-	if (self.window ~= nil and self.window.selected == self and not self.window.minimized and not self.window:GetWantsUpdates() and self.showSendChat) then
+	if (self.window ~= nil and self.window.selected == self and not self.window.minimized and not self.window:GetWantsUpdates() and self.showSendChat and self.window:IsVisible()) then
 		self.panel.chatSendWindow:SetSize(100,19);

```

Comment:
this if is true if the plugin is hidden, resulting in invisible button being put on user screen, 
Plugin open:
![ScreenShot_2019-11-23_220142_0](https://user-images.githubusercontent.com/5854320/69485304-d191ab00-0e3d-11ea-848e-1a874d3807d6.jpg)
Plugin hidden:
![ScreenShot_2019-11-23_220242_0](https://user-images.githubusercontent.com/5854320/69485305-d22a4180-0e3d-11ea-8955-1b020e6bf978.jpg)